keras_core/distribution/distribution_lib_test.py

```diff
@@ -28,7 +31,7 @@
 
 class DeviceMeshTest(testing.TestCase):
     def test_mesh_creation(self):
-        devices = ["CPU:{i}" for i in range(8)]
+        devices = [f"CPU:{i}" for i in range(8)]
```

Comment:
Thanks for catching this, we might want to fix this for test_input_validation as well.